gnome-initial-setup/gis-driver.c

```diff
@@ -187,6 +189,13 @@ get_image_version (void)
   return image_version;
 }
 
+const gchar *
+gis_driver_get_image_version (GisDriver *driver)
```

Comment:
I think this would be better if the driver split the image ID at the first `-`, stored that as `priv->product`, and then had:

`gboolean gis_driver_is_product (GisDriver *, const char *)` for a reason that will become clear…